### PR TITLE
Mirgate Terminal Setup Steps into Developer Guide

### PR DESCRIPTION
**Pull request by leonardozcm**
## Description

https://github.com/intel-analytics/BigDL/issues/3252#issuecomment-1188847383

### CODE REVIEW

**Comment by leonardozcm** [comment]
http://10.112.231.51:18888/job/BigDL-Orca-PR-Validation/829/

File: docs/readthedocs/source/doc/UserGuide/develop.md [hunk 1]
```diff
@@ -92,6 +92,24 @@ export PYTHONPATH=BigDL/python/dist/conf/spark-bigdl.conf:$PYTHONPATH
 The above environment variables should be available when running or debugging code in the IDE. When running applications in PyCharm, you can add runtime environment variables by clicking  __Run__ -> __Edit Configurations__; then in the __Run/Debug Configurations__ panel, you can add necessary environment variables to your applications.
 
 
+#### **1.3 Terminal Setup**
+
+Alternatively, for linux user you can set up environment variables mentioned above mantually, and we also provide a solution by following simple steps:
```
Comment (hkvision) [inline docs/readthedocs/source/doc/UserGuide/develop.md:97]: Besides setting the environment variables mentioned above manually for Linux users, we also provide a solution to set them with a script.

File: docs/readthedocs/source/doc/UserGuide/develop.md [hunk 2]
```diff
@@ -92,6 +92,24 @@ export PYTHONPATH=BigDL/python/dist/conf/spark-bigdl.conf:$PYTHONPATH
...
+
+```bash
+cd BigDL/python/orca
+source dev/prepare_env.sh
+```
+
+You can verify whether environment variables are available:
```
Comment (hkvision) [inline docs/readthedocs/source/doc/UserGuide/develop.md:104]: You can verify the BigDL environment by running the following example.
Maybe also guide users to check the environment variables manually?

File: docs/readthedocs/source/doc/UserGuide/develop.md [hunk 3]
```diff
@@ -92,6 +92,24 @@ export PYTHONPATH=BigDL/python/dist/conf/spark-bigdl.conf:$PYTHONPATH
...
+
+```bash
+pytest test/bigdl/orca/learn/ray/pytorch/test_estimator_pyspark_backend.py
+```
+
+Note that this approach will only work temporarily for terminal this time.
```
Comment (hkvision) [inline docs/readthedocs/source/doc/UserGuide/develop.md:110]: Change to `for this terminal`?

File: docs/readthedocs/source/doc/UserGuide/develop.md (hunk 2 above)
Comment (leonardozcm) [reply docs/readthedocs/source/doc/UserGuide/develop.md:104]: > 

Yes, in that case they can take examples in 1.2 as a reference.

Comment (leonardozcm) [reply docs/readthedocs/source/doc/UserGuide/develop.md:104]: > You can verify the BigDL environment by running the following example. Maybe also guide users to check the environment variables manually?

Yes, in that case they can take examples in 1.2 as a reference.

**Comment by hkvision** [comment]
Anywhere you mentioned users need to pip install bigdl-tf and math?

File: docs/readthedocs/source/doc/UserGuide/develop.md [hunk 4]
```diff
@@ -89,9 +89,40 @@ export PYTHONPATH=BigDL/python/dllib/src:BigDL/python/nano/src:BigDL/python/orca
...
+pip install bigdl-tf bigdl-math
+
+cd BigDL/python/friesian
+source dev/prepare_env.sh
+```
+
+You can verify the BigDL environment by running the following example.
+
+```bash
+python python/dllib/examples/autograd/custom.py
```
Comment (hkvision) [inline docs/readthedocs/source/doc/UserGuide/develop.md:120]: the path should be BigDL/python?

**Review by hkvision — APPROVED** [review]

**Pull request merged**